pom.xml

```diff
@@ -24,18 +24,21 @@
   <scm>
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-    <tag>HEAD</tag>
+    <tag>${scmTag}</tag>
   </scm>
 
   <properties>
-    <jenkins.version>1.651.3</jenkins.version>
+    <revision>1.9</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <jenkins.version>2.121.2</jenkins.version>
```

Comment:
Big jump, but required for JEP-210 compatibility. Based on [these usage stats](http://stats.jenkins.io/pluginversions/ant.html), around 73% of users of ant 1.8 are on Jenkins 2.121.2 or newer anyway, so this seems fine.

Reply:
Anyway even if there is some unrelated fix we realize we need to deliver urgently to the remaining 27%, it is easy enough to publish a backport.